log-processing/README.md

```diff
@@ -0,0 +1,60 @@
+ - Store full, raw logs in inexpensive storage (e.g., S3) for compliance, deep-dive, and batch analytics
+ - Query logs directly on compute nodes for immediate troubleshooting
+ - Scale quickly on multiple nodes, either on-prem or across different clouds, with minimal manual intervention
+
+**Key Advantages**:
+ - Substantial bandwidth and cost savings
+ - Flexible ingestion pipelines — use your favorite logging stack or data warehouse
+ - Simplified troubleshooting with local or near-local query options
+ - Straightforward job orchestration via Bacalhau’s CLI or APIs
+
+## Run This Example
+
+### Prerequisites
+- Docker and Docker Compose
+- [Expanso Cloud](https://cloud.expanso.io) account
+
+### 1. Create an Orchestrator
+Use [Expanso Cloud](https://www.cloud.expanso.io/) to create a new Bacalhau network (a managed orchestrator node). Name it however you like.
+
+### 2. Spin Up and Connect Compute Node
+1. Under _"+ Node"_ in your new network, obtain the token and NATS URL from the platform.
+2. In this directory, update `compute.yaml` with the relevant token and NATS URL replacing all instances of `<TODO>`. If necessary, change the number of compute nodes you want to run under `services.compute.deploy.replicas` in `docker-compose.yml`.
+
+3. Start the environment:
+   ```bash
+   docker compose up
+   ```
+   This command will spin up Bacalhau compute nodes, a storage (MinIO) node and OpenSearch nodes.
+
+### 3. Generate Logs
+In the Expanso Cloud dashboard, go to **Network > _YourNetwork_ > Jobs**, create a new job from file, use `generate-logs.yaml` and run it. This [daemon](https://docs.bacalhau.org/cli-api/specifications/job/type#daemon-jobs) job will run on all compute nodes and generate logs for this example. Once submitted you should be redirected to the Job page. Here you can see its status, history and executions. Under the executions tab you should see one execution per node.
+
+### 4. Run the Logstash Job
+In Expanso Cloud, go to **Network > _YourNetwork_ > Jobs**, create or upload the job `logstash.yaml` and run it. This is another [daemon](https://docs.bacalhau.org/cli-api/specifications/job/type#daemon-jobs) job that will collect logs, writing aggregated logs to OpenSearch, and storing raw logs in MinIO. After a few minutes, verify logs are being collected by checking your logs bucket at `localhost:9001` (username and password configured in `docker-compose.yml`) and your OpenSearch dashboard at `localhost:5601`.
+
+### 5. Query Logs directly on Compute Node
+In cases where you need to query logs immediately due to an incident or alert, Bacalhau [ops](https://docs.bacalhau.org/cli-api/specifications/job/type#ops-jobs) jobs can help you filter logs on the compute node itself since recent logs might not be available in the centralized storage yet.
+
+Upload and run the job `query-logs.yaml` to filter logs for specific HTTP statuses (for example, 404 errors) right on the compute nodes. The results will be stored in MinIO.
```

Comment:
I don't think we need to store the query result in minio. Maybe we can say it as a bonus, but retiving the results should be easier thant going to minio, or calling `bacalhau jog get`, which may not work since the orchestrator doesn't have access to minio to sign the request

Reply:
I didn't include that since `bacalhau job get` doesn't return any results and `bacalhau job logs` returns an error since v1.7.

```
❯ bac job get j-9947c2a9
Fetching results of job 'j-9947c2a9'...
No results found
You can check the logged output of the job using the logs command.

  bacalhau job logs j-9947c2a9
❯ bacalhau job logs j-9947c2a9

Error: unknown error trying to stream logs from job (ID: j-9947c2a9): websocket: bad handshake
```

Still have to debug this

Reply:
Getting same error:
```
❯ bac job logs j-fc09ff2e-b1c1-4219-8dd9-af07be841646

Error: unknown error trying to stream logs from job (ID: j-fc09ff2e-b1c1-4219-8dd9-af07be841646): websocket: bad handshake
```